src/db/db.ts

```diff
@@ -0,0 +1,35 @@
+import mongoose from 'mongoose';
+import config from 'config';
+
+mongoose.Promise = global.Promise;
```

Comment:
mongoose does not support native promises?

Reply:
Looks like it's legacy code from Mongoose 4 (We're using 5). I'll remove it